[REF] Funcionalidade de template de informações adicionais

lfdivino opened the pull request:
- Generalizada a funcionalidade para renderizar as informações
complementares dos documentos fiscais. Cada documento vai
preencher a sua maneira as informações complementares e assim
chamar a função para renderizar o template dessas informações.

lfdivino committed:
[REF] Funcionalidade de template de informações adicionais

- Generalizada a funcionalidade para renderizar as informações
complementares dos documentos fiscais. Cada documento vai
preencher a sua maneira as informações complementares e assim
chamar a função para renderizar o template dessas informações.

mileo reviewed (changes requested):
O try deve ficar dentro do método de renderização e fazer parte de todos os escopos

lfdivino committed:
[ADD] Informações complementares produto CFe

- Montar as informações complementares do produto do CFe e
renderizar o template seguindo o modelo da NFe

lfdivino committed:
[FIX] Validação do Try no renderizar do template

lfdivino commented:
Alterado o processamento dos Try.
